subsys/bluetooth/services/ots/Kconfig

```diff
@@ -69,6 +59,46 @@ config BT_OTS_OLCP_GO_TO_SUPPORT
 	bool "Support OLCP Go To Operation"
 	default y
 
+config BT_OTS_OBJ_NAME_WRITE_SUPPORT
+	bool "Support object name write"
+
+endif # BT_OTS
+
+#### Object Transfer Service Client ################################
+
+config BT_OTS_CLIENT
+	bool "Object Transfer Service Client [Experimental]"
+	select BT_L2CAP_DYNAMIC_CHANNEL
+	select BT_GATT_CLIENT
+	select BT_GATT_AUTO_DISCOVER_CCC
+	select EXPERIMENTAL
+	default n
+	help
+	  This option enables support for the Object Transfer Service Client.
+
+if BT_OTS_CLIENT
+
+# TODO: Unify logging for OTS server and client
```

Comment:
This may need to be more specific, as this could also mean that you'd only want a single Kconfig to enable both.

Reply:
Fixed